tap_gsheets/__init__.py

```diff
@@ -100,6 +110,15 @@ def process_worksheet(gsheets_loader, sheet_name, worksheet, start_from_row, con
         singer.write_record(stream_name, record_transformed)
 
 
+def non_standard_date_execution(records):
+    for record in records:
+        for field in record:
```

Comment:
So, basically this is looping through every record and every field trying to parse the date, right?
How many records we have in there?

I am just a bit concerned, because this could be quite impactful in the performance.

Reply:
As an alternative, it should be possible to look for a date column using only the first record. Would that work?

Reply:
I'll think about it. i believe that's this way incorrect. All date fields will be transformed to date_format. need to change whole logic